certbot-dns-cloudxns/local-oldest-requirements.txt

```diff
@@ -1,2 +1,2 @@
-acme[dev]==0.21.1
-certbot[dev]==0.21.1
+acme[dev]==0.25.0
```

Comment:
I updated it because to use `acme.magic_typing` in `certbot.plugins.dns_common_lexicon`, `acme>=0.25.0` is required. So it is not to update against Certbot in general, but only for Lexicon-based DNS plugins, that will rely on `certbot.plugins.dns_common_lexicon`. This change is also reflected in the relevant `setup.py` of course.

I did that to avoid to depend unnecessarily to too recent version of `acme`, only to the minimal one in the local oldest spirit. It avoid also the risk that you exposed in solution 1.

However if you prefer, we can go to solution 2., and put `-e acme` as local oldest.

Reply:
Ok. I update to latest dev version